config/templates/hive-csv-template.yaml

```diff
@@ -4,56 +4,108 @@ metadata:
   name: hive-operator-0.0.1
   namespace: placeholder
   annotations:
-    categories: A list of comma separated categories that your operator falls under.
+    capabilities: Seamless Upgrades
+    categories: OpenShift Optional
     certified: "false"
     description: OpenShift cluster provisioning and management at scale.
-    containerImage: quay.io/dgoodwin/hive:latest
+    containerImage: quay.io/openshift-hive/hive:v0.0.1
     createdAt: "2019-02-25T14:29:00Z"
-    support: Devan Goodwin
+    repository: https://github.com/openshift/hive
+    support: Hive Team
+    alm-examples: |-
+      [{"apiVersion":"hive.openshift.io/v1","kind":"HiveConfig","metadata":{"name":"hive"},"spec":{"managedDomains":[{"aws":{"credentialsSecretRef":{"name":"my-route53-creds"}},"domains":["my-base-domain.example.com"]}]}}]
 spec:
-  displayName: Hive
-  description: OpenShift cluster provisioning and management at scale.
+  displayName: Hive for Red Hat OpenShift
```

Comment:
"Hive for OpenShift Container Platform" or just "OpenShift Hive" would feel better than "Red Hat OpenShift" to me, wdyt?

Reply:
this is a request from Brand. it must remain :)